### DISCUSSION TITLE
Unable to convert .ipynb file to .html file

### DISCUSSION THREAD

**Discussion by oye-bobs** [cu-esiil-edu/2024-data-short-course#89] [Q&A]
@nquarder 
I tried to convert my vegetation notebook to an html file, but was unable to. I then proceeded to convert it to a markdown file. 

On doing this I tried to upload it to my porfolio page but the build failed, and tried to link it with my badge collection markdown file and it still failed

How can I fix this problem?

**Comment by nquarder** [comment]
Hi @oye-bobs - I can see your notebook and md files for the analysis at Gwagwalada. Nice work! 

![image](https://github.com/cu-esiil-edu/2024-data-short-course/assets/16786791/c796c5c6-81d1-4742-81f8-f4d39feb5b56)

![image](https://github.com/cu-esiil-edu/2024-data-short-course/assets/16786791/21f88e02-fd34-466f-9ac9-f8ab3f6f78e0)


I do see that you have an .html of your time-series analysis at Murtala in your notebooks folder in your oye-bobs.github.io repository. 

https://oye-bobs.github.io/notebooks/murtala_time_series_2.html

Maybe try again to convert the vegetation notebook to `.html` and try adding that to your notebooks folder?

One thing I'm noticing at the bottom of your vegetation notebook is the following code to convert from `ipynb` to `html`:

```
!jupyter nbconvert *.ipynb --to html
```

When I convert from `ipynb` to `html` I use the following lines of code that include the `%%capture` and `%%bash` statements and does not include the `!` in front of `jupyter` like you have in your code. 

See about replacing your code with what I have below and try again to convert to `html`. 

```
%%capture
%%bash
jupyter nbconvert *.ipynb --to html

```

Give this a try and see how it goes. -NQ

**Reply by oye-bobs** [reply]
Hi @nquarder I changed that line of code and got this error :

![10](https://github.com/cu-esiil-edu/2024-data-short-course/assets/33554816/b16bccc0-b5ed-4031-8cd1-6f46b8b5ec43)
![11](https://github.com/cu-esiil-edu/2024-data-short-course/assets/33554816/6bf56fd1-4cb2-47b9-ae6a-ccbbcc9771b0)

**Comment by nquarder** [comment]
okay @oye-bobs i'm trying to get into your repository now to take a look at your notebook. i'm creating a codespace and will try to do the analysis for Abuja to see about recreating your error. I'll let you know what I find out.  -NQ

**Comment by nquarder** [comment]
Hi again @oye-bobs - I was able to open your vegetation notebook in a codespace and am also getting the same error you are when converting to html. I'm not sure what is happening there but I did change back to markdown and re-ran. The markdown seems to print okay. You might try creating a entirely new notebook for the Abuja analysis and try doing the convert to html again? Your analysis is looking good! -NQ

**Comment by oye-bobs** [comment]
@nquarder , Alright I'll do that.

**Reply by oye-bobs** [reply]
Hi @nquarder, I tried recreating a new notebook, but ran into the same error when trying to convert from .ipynb to .html.

**Comment by nquarder** [comment]
Okay @oye-bobs - it looks like you were able to get your issue figured out? 

https://oye-bobs.github.io/notebooks/gwagwalada.html

Nice job with this analysis of Gwagwalada. -NQ
